### PR TITLE
DOC for Netconf test tool docker image

### PR DESCRIPTION
**Pull request by ltokarcik**
- DOC for running Netconf test tool from docker image on Docker hub

JIRA: UNIC-1159

### CODE REVIEW

File: frinx-uniconfig/user-guide/network-management-protocols/uniconfig_netconf/readme.md [hunk 1]
```diff
@@ -964,6 +965,90 @@ Notes:
...
+-s or --schemas to set schemas folder (required firstly mount schemas folder to docker container using -v)
+-i or --init-conf to set initial simulated config to be returned by get-config RPC (required firstly mount this config file to docker container using -v)
+--ssh to enable ssh
+--md-sal to enable md-sal
+--debug to run in debug mode
+-h or --help to display help options
+--override to override these options to access all netconf test tool options directly
+```
+**note**  
+To set value of xmx is required to enter xmx heap size value separated from -x or --xmx option.
```
Comment (stanislav-chlebec) [inline frinx-uniconfig/user-guide/network-management-protocols/uniconfig_netconf/readme.md:992]: i do not quite understand.
what does mean separated in the context of all sentence?
ahhh i remember
you mean there must be a space between keyword and value.
please rephrase e.g.
while using xmx option you must use space before entered value

File: frinx-uniconfig/user-guide/network-management-protocols/uniconfig_netconf/readme.md [hunk 2]
```diff
@@ -964,6 +965,90 @@ Notes:
...
+4. you can also use help option to show how to use it using -h or --help
+```
+docker run --rm -it frinx/netconf-testtool --help
+```
+5. you can also override simple interface of starting script and use all options of test tool directly
+```
+docker run --publish 1024-1123:1024-1123 -v ./schemas:/opt/schemas -v ./i.xml:/opt/i.xml -v ./notification.xml:/opt/notification.xml \
+            -it frinx/netconf-testtool --xmx 1G --override \
+            --ssh true \
+            --md-sal true
```
Comment (stanislav-chlebec) [inline frinx-uniconfig/user-guide/network-management-protocols/uniconfig_netconf/readme.md:1015]: please add missing backslash at the end

File: frinx-uniconfig/user-guide/network-management-protocols/uniconfig_netconf/readme.md [hunk 3]
```diff
@@ -964,6 +965,90 @@ Notes:
...
+**note**  
+If you override starting script options, you should set Java heap size using -x or --xmx option before --override option.  
+You could not use any other starting script options except xmx and debug option.  
+If you override starting script options, --debug option activate only Java and shell debug mode, but not debug mode of test tool itself.  
+If you want to start debug mode of netconf test tool, you should use debug options of Netconf test tool directly.  
+
+6. you can also override docker entrypoint and run netconf test tool by custom way using all options including java options and run like bellow
+```
+docker run --publish 1024-1123:1024-1123 -v ./schemas:/opt/schemas -v ./i.xml:/opt/i.xml -v ./notifications.xml:/opt/notification.xml \
+-it --entrypoint /bin/bash frinx/netconf-testtool
```
Comment (stanislav-chlebec) [inline frinx-uniconfig/user-guide/network-management-protocols/uniconfig_netconf/readme.md:1032]: please add backslash

File: frinx-uniconfig/user-guide/network-management-protocols/uniconfig_netconf/readme.md [hunk 4]
```diff
@@ -964,6 +965,90 @@ Notes:
...
+
```
Comment (stanislav-chlebec) [inline frinx-uniconfig/user-guide/network-management-protocols/uniconfig_netconf/readme.md:1033]: please add backslash

**Review by stanislav-chlebec — CHANGES_REQUESTED** [review]

**Review by marosmars — APPROVED** [review]

**Pull request merged**